js/renovation/utils/plugin/placeholder_item.tsx

```diff
@@ -46,8 +51,39 @@ export class PlaceholderItemProps {
 
 @Component({ defaultOptionRules: null, view: viewFunction })
 export class PlaceholderItem extends JSXComponent(PlaceholderItemProps) {
+  @Consumer(PluginsContext)
+  plugins!: Plugins;
+
+  @InternalState()
+  updateCounter = 0;
+
+  @Effect()
+  updateArgs(): () => void {
+    const disposers = this.componentDeps.map((entity) => this.plugins.watch(entity, () => {
+      this.updateCounter += 1;
+    }));
```

Comment:
Maybe it's better to use something like

```ts
@InternalState()
args: Record<number, unknown> = {}

@Effect()
updateArgs(): () => void {
  const disposers = this.componentDeps.map((entity) => this.plugins.watch(entity, (value) => {
  this.args = {...this.args, [entity.id]: value}
}));

get args(): unknown[]
  return this.componentDeps.map(
    (entity) => this.args[entity.id],
  );
}
```

The problem with memorized getter can be resolved using it